tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_SimpleWenoImpl.cpp

```diff
@@ -373,12 +389,28 @@ void test_simple_weno_2d(const std::unordered_set<Direction<2>>&
                            neighbor_modified_vars);
 }
 
-void test_simple_weno_3d(const std::unordered_set<Direction<3>>&
-                             directions_of_external_boundaries = {}) noexcept {
-  INFO("Test simple_weno_impl in 3D");
+void test_simple_weno_2d() noexcept {
+  INFO("Test simple_weno_impl in 2D");
+  const auto gl = Spectral::Quadrature::GaussLobatto;
+  const auto gauss = Spectral::Quadrature::Gauss;
+
+  test_simple_weno_2d_impl(gl);
+  test_simple_weno_2d_impl(gauss);
+
+  // Test with particular boundaries labeled as external
+  test_simple_weno_2d_impl(gl, {{Direction<2>::lower_eta()}});
```

Comment:
maybe either indicate that gauss points don't work in this case or test gauss points too?